src/init.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
This file will be distributed in our npm module, and provides functions for retrieving the manual initialization function and registry.